lib/computer_player.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './player.rb'
+
+class ComputerPlayer < Player
```

Comment:
Code Smell: Refused Bequest.  Inheritance really serves no purpose here. The parent has no methods that this subclass relies on.